lib/sidekiq/profiler.rb

```diff
@@ -13,6 +13,7 @@ class Profiler
     include Sidekiq::Component
     def initialize(config)
       @config = config
+      @profiles_dir = ENV.fetch('PROFILES_DIR') { Dir.tmpdir }
```

Comment:
Everything else looks solid but I'm not sure about the ENV name. Generally I think it's a good idea to prefix the variable with the intended recipient, e.g. `VERNIER_OUTPUT_DIR`. That would also allow Vernier to adopt the variable internally too. See https://github.com/jhawthorn/vernier/issues/146

Reply:
No problem. Updated